docs/argo-server-sso.md

```diff
@@ -112,6 +112,44 @@ If no rule matches, we deny the user access.
 
     The precedence must be the lowest of all your service accounts.
 
+## SSO RBAC Namespace Delegation
+
+You can optionally configure RBAC SSO per namespace.
+Typically, on organization has a K8s cluster and a central team manages the cluster who is the owner of the cluster. Along with this, there are multiple namespaces which are owned by individual team. This feature would help namespace owners to define RBAC for their own namespace.
+
+The feature is currently in beta.
+To enable the feature, set env variable `SSO_DELEGATE_RBAC_TO_NAMESPACE=true` in your argo-server deployment.
+
+#### Recommended usage
+
+Configure a default account in the installation namespace which would allow all users of your organization. We will use this service account to allow a user to login to the cluster.
```

Comment:
Should we note that this does not need any role or role binding?

Reply:
The role and rolebinding would be optional.
They can associate a read-only role. Since, if no account in namespace matches, we will be using the default role for the operation

Reply:
Let me add that